src/pages/home/report/ContextMenu/ContextMenuActions.tsx

```diff
@@ -180,7 +180,7 @@ const ContextMenuActions: ContextMenuAction[] = [
     {
         isAnonymousAction: false,
         textTranslateKey: 'reportActionContextMenu.replyInThread',
-        icon: Expensicons.ChatBubbleAdd,
```

Comment:
Since we aren't using `ChatBuubleAdd` any more, should we remove it from Expensicons.ts and assets as well? Or will we be using it in some other place?

Reply:
I suspect we'll use it later. As we still have it in our Figma library I'd be keen to keep it for now :)